src/app/route/CheckoutPage/CheckoutPage.container.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @link https://github.com/scandipwa/base-theme
+ */
+
+import { connect } from 'react-redux';
+import { HeaderAndFooterDispatcher } from 'Store/HeaderAndFooter';
+import CheckoutPage from './CheckoutPage.component';
+
+const mapStateToProps = state => ({
+    products: state.CartReducer.products,
```

Comment:
The Cart reducer has changed it's state key for products, please pull `my-account-frontend`.